nmcli: amended the routing-rules4 key values as list

sree0744 opened the pull request:
##### SUMMARY
Amended the routing-rules4 values as list

<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->

##### ISSUE TYPE ######

- Feature Pull Request

##### COMPONENT NAME
nmcli
 
##### ADDITIONAL INFORMATION #########

- scope - routing_rules4 module argument is currently accepting only string elements. In the case of adding multiple entries to routing_rules4, we need to accept values as strings.
(https://github.com/ansible-collections/community.general/pull/3385).

sree0744 reviewed (commented):
Fixed Typo line #1701

felixfontein reviewed (commented):
Thanks for your contribution! Please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to). It would also be great if you could update the tests in tests/unit/plugins/modules/net_tools/test_nmcli.py.

felixfontein commented:
> Fixed Typo line #1701

Not sure what you mean here? PR #1701 is not related to this change as far as I can tell.

sree0744 commented:
Thanks @felixfontein for the review. I've updated the changes under tests/unit/plugins/modules/net_tools/test_nmcli.py. Also added changelog fragment.

- scope - routing_rules4 module argument is currently accepting only string elements. In the case of adding multiple entries to routing_rules4, we need to accept values as strings.
(https://github.com/ansible-collections/community.general/pull/3385).

sree0744 commented:
Thanks @felixfontein. I tried ran some unit tests with nmlci.py code(that i've updated. It seems no luck to get results. Could you please refer or help further on this please.

i've tried to run multiple times -  ansible-test units tests/unit/plugins/modules/net_tools/test_nmcli.py --docker -v

Thanks

sree0744 commented:
Closing this PR. Will create new PR with tests output
